core/blockchain_test.go

```diff
@@ -18,57 +18,22 @@ package core
 
```

Comment:
`blockchain.go` has significant amount of changes surrounding the newly added trading&lending features, but lack of tests covered in this file.